### DISCUSSION TITLE
New mono depthmap algos

### DISCUSSION THREAD

**Discussion by graemeniedermayer** [thygate/stable-diffusion-webui-depthmap-script#470] [Ideas]
Apple made a monodepth. It has a weird license so I'm not sure about integration, but it does seem impressive/high quality.

https://github.com/apple/ml-depth-pro

also Metric3dv2 seems to have very good metrics. (I might work on an integration)

https://jugghm.github.io/Metric3Dv2/

**Comment by vdruts** [comment]
Came here to post this :)

**Comment by sonakravtsenko3** [comment]
Another step forward for depth estimation models! Looks impressive.

**Comment by graemeniedermayer** [comment]
https://github.com/microsoft/MoGe
another one!
